app/bundles/InstallBundle/EventListener/DoctrineEventSubscriber.php

```diff
@@ -18,13 +19,18 @@ public function getSubscribedEvents()
 
     public function postGenerateSchema(GenerateSchemaEventArgs $args): void
     {
-        $schema = $args->getSchema();
-
-        $fieldGroups['leads']     = FieldModel::$coreFields;
-        $fieldGroups['companies'] = FieldModel::$coreCompanyFields;
+        $fieldGroups = [
+            'leads'     => FieldModel::$coreFields,
+            'companies' => FieldModel::$coreCompanyFields,
+        ];
 
         foreach ($fieldGroups as $tableName => $fields) {
-            $table = $schema->getTable(MAUTIC_TABLE_PREFIX.$tableName);
+            try {
+                $table = $args->getSchema()->getTable(MAUTIC_TABLE_PREFIX.$tableName);
+            } catch (SchemaException $e) {
+                // Ignore during plugin installations as not all tables are present in the schema.
+                continue;
+            }
```

Comment:
@mollux do you mean like this?
```suggestion
            $fullTableName = MAUTIC_TABLE_PREFIX.$tableName;
            if (!$args->getSchema()->hasTable($fullTableName)) {
                // Ignore during plugin installations as not all tables are present in the schema.
                continue;
            }
            $table = $args->getSchema()->getTable($fullTableName);
```
I was thinking about that but decided to do not introduce one more variable. Do you suggest that because throwing and catching the exception is a split millisecond slower than checking for the table?

Reply:
No, not for the speed impact, more the eating up an error that could be useful. We know the exact conditions (table should exist), so we can check that. If an error is thrown, something else is wrong. Just my 2 cents :)

Reply:
That makes total sense. I appreciate your $0.02! I'll commit the suggestion I made above.